markdown.json

```diff
@@ -0,0 +1,57 @@
+{
+  "#": [
+    "Markdown is a shorthand for creating HTML from text strings."
+  ],
+  "methods": {
+    "description": [
+      "Check the public API is correct."
+    ],
+    "cases": [{
+      "description": "must be able to parse a Markdown string",
+      "method": "parse",
+      "arity": 1
+    }]
+  },
+  "cases": [
+    {
+      "description": "parsing italics",
+      "input": "_This will be italic_",
+      "expected": "<p><i>This will be italic</i></p>"
+    },
+    {
+      "description": "parsing bold text",
+      "input": "__This will be bold__",
+      "expected": "<p><em>This will be bold</em></p>"
+    },
+    {
+      "description": "mixed normal, italics and bold text",
+      "input": "This will _be_ __mixed__",
+      "expected": "<p>This will <i>be</i> <em>mixed</em></p>"
+    },
+    {
+      "description": "with h1 header level",
+      "input": "# This will be an h1",
+      "expected": "<h1>This will be an h1</h1>"
+    },
+    {
+      "description": "with h2 header level",
+      "input": "## This will be an h2",
+      "expected": "<h2>This will be an h2</h2>"
+    },
+    {
+      "description": "with h6 header level",
+      "input": "###### This will be an h6",
+      "expected": "<h6>This will be an h6</h6>"
+    },
+    {
+      "description": "unordered lists",
+      "input": "* Item 1\n* Item2",
```

Comment:
Missing a space at the end of line 48 between `Item` and `2`, for anyone like me who was copying/pasting samples!